ack/readers/awin_advertiser/cli.py

```diff
@@ -28,7 +28,9 @@
 @click.option("--awin-advertiser-id", required=True)
 @click.option("--awin-report-type", type=click.Choice(REPORT_TYPES), default=REPORT_TYPES[0])
 @click.option("--awin-region", required=True)
+@click.option("--awin-campaign")
 @click.option("--awin-timezone", required=True)
+@click.option("--awin-interval")
```

Comment:
If they are not required, you should put a default value

Reply:
have added default=None to each